js-exercises/aperture/aperture.js

```diff
@@ -0,0 +1,13 @@
+function aperture(tuple, array) {
```

Comment:
tuple is a sequence of elements, but here we are expecting the tuple length to be sent , so tupleLength would be a good name to convey the idea of method.